test/Base_Tests/src/Data/Polyglot_Spec.enso

```diff
@@ -34,6 +34,14 @@ add_specs suite_builder = suite_builder.group "Polyglot" group_builder->
     group_builder.specify "access Integer field from Polyglot object" <|
         js_meaning.meaning . should_equal 42
 
+    group_builder.specify "Execute JavaScript function" <|
+        js_plus 3 5 . should_equal 8
+
+    group_builder.specify "Cannot Execute JavaScript number" <|
+        fourty_two = js_meaning.meaning
+        Test.expect_panic Not_Invokable <|
+            fourty_two "Cannot invoke"
```

Comment:
I don't think I understand this. Intuitively, this should not try to _invoke_ the number, it should just access the field. How does this test differ from `"access Integer field from Polyglot object"` above? IMO they both are the same - whether the result is assigned to a variable or not should not matter here.

Reply:
No, sorry. I have completely misread the code snippet. Please ignore my above comment.